### ISSUE TITLE
OverSampling using RF

### ISSUE THREAD

**Issue by MattiaPOLI** [MattiaPOLI/DR14#2]
Due the fact that our dataset is unbalanced (low number of QSO's instances) it is interesting to use oversampling techniques (SMOTE) but:
- RF with bootstrap doesn't use the whole training set
- oob score is evaluated starting from data not used during the training phase (no need to separate training set/validation set)

While oversampling is it mandatory to first "remove" the validation set and then oversample, does it make any sense to use it in this scenario?
